### ISSUE TITLE
Accessibility: Unlabled control in settings pane.

### ISSUE THREAD

**Issue by TheQuinbox** [microsoft/terminal#13658]
### Windows Terminal version

Windows Terminal 1.14.1962.0

### Windows build number

Windows 10 21H2 (64-bit) build 19044.1826

### Other Software

None (in the settings section of Terminal).

### Steps to reproduce

1. Start narrator with Control+Windows+Enter
2. Open Windows Terminal
3. Open settings with control+,
4. Tab 3 times.

### Expected Behavior

The button should have an accessibility label, so I can tell what its intended to do.

### Actual Behavior

I just hear "button", or "button 1 of1", depending on which screen reader I use.

**Comment by zadjii-msft** [comment]
Is what you're encountering basically the same as #13020/?

**Comment by TheQuinbox** [comment]
Hi @zadjii-msft,
No, I do not have the issue outlined in #13020. In fact, I'm not able to reproduce it. My focus gets landed on the open/close navigation button when first opening preferences. This is honestly a fairly small issue. The button next to the open settings.json file doesn't have an accessibility label, so I don't know what it is. I can tab to it, but it just doesn't have a name that my screen reader can see.

**Comment by carlos-zamora** [comment]
Hey! Looks like this ended up getting fixed by #14180. Let us know if you find any other ones. Sorry for taking a while to get back to you.
